content/docs/tasks/telemetry/distributed-tracing/index.md

```diff
@@ -61,11 +61,11 @@ Although every service has the same label, `istio-proxy`, because the tracing is
 the Istio sidecar (Envoy proxy) which wraps the call to the actual service,
 the label of the destination (to the right) identifies the service for which the time is represented by each line.
 
-The first line represents the external call to the `productpage` service. The label `192.168.64.3:32000` is the host
+The first line represents the external call to the `productpage` service. The label `192.168.39.95:31380` is the host
 value used for the external request (i.e., $GATEWAY_URL). As you can see in the trace,
-the request took a total of roughly 290ms to complete. During its execution, the `productpage` called the `details` service,
-which took about 24ms, and then called the `reviews` service.
-The `reviews` service took about 243ms to execute, including a 15ms call to `ratings`.
+the request took a total of roughly 45.56ms to complete. During its execution, the `productpage` called the `details` service,
+which took about 1.61ms, and then called the `reviews` service.
+The `reviews` service took about 36.44ms to execute, including a 1.1ms call to `ratings`.
```

Comment:
Do you think we should mention the client and server spans here?

Reply:
Thanks..Fixed